libkbfs/tlf_handle.go

```diff
@@ -126,12 +127,22 @@ func MakeBareTlfHandle(
 		sort.Sort(SocialAssertionList(unresolvedReadersCopy))
 	}
 
+	var conflictInfo, finalizedInfo *TlfHandleExtension
+	for _, extension := range extensions {
```

Comment:
Is there anything dangerous in the fact that a user can type either "(conflicted...) (finalized...)" and "(finalized...) (conflicted...)" on the command line and end up with the same TLF, because this function doesn't make sure the extensions are in the usual canonical order?  I can't think of anything, except possibly that if the user has both paths to the TLF active, the kernel could get confused about its cached when data is updated via one path.

It might avoid some confusion if we either enforce an order here, and/or return a non-canonical error from the TLF parsing function, to turn the unusual version into a symlink.